File: lib/otel_sample/otel_sample.ex
defmodule OtelSample do
  @external_resource "docs/otel_sample.md"
  @moduledoc TemporalSdk.Utils.Code.exdoc!("docs/otel_sample.md")

  def run, do: TemporalSdk.start_workflow(:cluster_1, "default", OtelSample.Workflow)
end

File: src/otel_sample/otel_sample.erl
-module(otel_sample).

% elp:ignore W0012 W0040 E1599
-moduledoc {file, "../../docs/otel_sample.md"}.

-export([
    run/0
]).

run() -> temporal_sdk:start_workflow(cluster_1, "default", otel_sample_workflow).
